superset/translations/de/LC_MESSAGES/messages.json

```diff
@@ -284,6 +283,9 @@
       "A valid color scheme is required": [
         "Ein gültiges Farbschema ist erforderlich"
       ],
+      "A waterfall chart is a form of data visualization that helps in understanding\n          the cumulative effect of sequentially introduced positive or negative values.\n          These intermediate values can either be time based or category based.": [
```

Comment:
Why do we need newlines and spaces here?

Reply:
@john-bodley could you please move this question to #26074? @Ralkion can help you with this.

Reply:
The short answer is that this is how it's built in the code. The first entry of each array is the 'key' that is used for lookup to get the 'value'.

In this case, you can trace the key via the messages.pot file (superset/translations/messages.pot), which is a generated file. The pot file has a comment indicating where the string originated - which for this message is (superset-frontend/plugins/plugin-chart-echarts/src/Waterfall/index.ts:53)